src/controllers/initgamecontroller.js

```diff
@@ -31,7 +31,10 @@ class InitGameController extends Controller
         );
 
         $(document).ready(function () {
-            HelpController.startIntroTour();
+            if (!Cookies.get('hasVisited')) {
+                HelpController.startNavbarTour(true);
+                Cookies.set('hasVisited', true, {expires: 7});
+            }
```

Comment:
This wasn't working for me.
Might have been some error from my side.
If it does work, then this is of course better :)